utils/dev/run-server.sh

```diff
@@ -8,6 +8,9 @@ command -V mc
 
 projRoot=`git rev-parse --show-toplevel`
 
+cd $projRoot/server/server
+cargo check
+
 dir="$1"
```

Comment:
How would you feel about making the directory an env variable and cargo args `@`?